### DISCUSSION TITLE
Social links in user sheet are trimmed after the first occurrence of `/` after domain name

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#60383] [Mobile] [answered]
### Select Topic Area

Bug

### Body

If the user have an URL like `https://example.com/foo/bar`, only text up to `foo` will display  
A great example is discord user profiles links :  
  
![Screenshot_2023-07-08-10-43-57-352_com github android](https://github.com/community/community/assets/82015596/0bfea9a9-5e06-4e59-bd15-200d1f26cac8)  
  
Note that clicking on it works perfectly and leads to the desired URL

**Comment by aconsuegra** [comment]
Hi @EDM115, thanks for reaching out. I'll send this to our engineering team and we'll try to find out what is wrong.

**Comment by EDM115** [comment, marked as answer]
Fixed in `v.1.123`
